Community/AssetManagement/README.md

```diff
@@ -0,0 +1,19 @@
+<!-- Copyright 2019 BlueCat Networks. All rights reserved. -->
+
+©2019 BlueCat Networks (USA) Inc. and its affiliates (collectively ‘ BlueCat’). All rights reserved. This document contains BlueCat confidential and proprietary information and is intended only for the person(s) to whom it is transmitted. Any reproduction of this document, in whole or in part, without the prior written consent of BlueCat is prohibited.
+
+Workflow Version: **1.0** <br/>
+Project Title: **BlueCat FireMon Integration** <br/>
+Author: **Lumeta/FireMon** <br/>
+Date: **DD-MM-YYYY** <br/>
+BlueCat Gateway Version: **19.8.1** <br/>
+BAM / BDDS Version: **8.3.0** <br/>
+Dependencies: **None** <br/>
+Installation Directions: **Description of installation steps** <br/>
+Known Errors and Bugs: **Short description of errors and bugs** <br/>
+Description/Example Usage: **This workflow allows you to retrieve a list of assets managed by BAM, retrieve a list of configurations that exist in BAM and also to add ip address to BAM. Depending on the flag in the payload, API creates network block and/or Address Block
+if containing network or address block does not exist. **<br/>
+BAM API methods: **/lumeta/getiplist
+/lumeta/getnetworklist
+/lumeta/addiplist** <br/>
```

Comment:
BAM methods are in reference to any BAM APIs invoked to help maintain compatibility if there's changes made to them down the road. Not absolutely essential but very helpful. For instance I see you are using .get_configurations(), get_object_by, etc.